src/app/app.module.ts

```diff
@@ -8,9 +8,11 @@ import { NotAuthorizedGuard } from '@app/auth/guards/not-authorized.guard';
 import { AuthorizedGuard } from '@app/auth/guards/authorized.guard';
 import { CoursesStoreService } from '@app/services/courses-store.service';
 import { CoursesService } from '@app/services/courses.service';
+import { CoursesComponent } from './features/courses/courses.component';
+import { CoursesListComponent } from './features/courses/courses-list/courses-list.component';
 
 @NgModule({
-  declarations: [AppComponent, CourseInfoComponent],
+  declarations: [AppComponent, CourseInfoComponent, CoursesComponent, CoursesListComponent],
```

Comment:
As per task requirements, you should add these components in their separate module (and distribute the module across the application)
![image](https://github.com/user-attachments/assets/36821bcd-f9ba-4733-8e36-22cdf6777a23)
(Obviously, it is `courses.module.ts` not `.scss`)